Lombiq.Tests.UI/Extensions/ApplicationLogEnumerableExtensions.cs

```diff
@@ -8,8 +8,25 @@ namespace Lombiq.Tests.UI.Extensions;
 
 public static class ApplicationLogEnumerableExtensions
 {
-    public static async Task<string> ToFormattedStringAsync(this IEnumerable<IApplicationLog> logs) =>
-        string.Join(
+    public static async Task<string> ToFormattedStringAsync(this IEnumerable<IApplicationLog> logs)
+    {
+        var logsArray = logs.ToArray();
+
+        if (logsArray.Length == 1)
+        {
+            return Environment.NewLine + await LogLinesToFormattedStringAsync(logsArray[0]);
+        }
+
+        return string.Join(
             Environment.NewLine + Environment.NewLine,
-            await Task.WhenAll(logs.Select(async log => log.Name + Environment.NewLine + Environment.NewLine + await log.GetContentAsync())));
+            await Task.WhenAll(
+                logsArray.Select(async log =>
+                    $"# Log name: {log.Name}" +
+                    Environment.NewLine +
+                    Environment.NewLine +
+                    await LogLinesToFormattedStringAsync(log))));
```

Comment:
This approach may change the order of the log records. I would be surprised if we will have more than a few hundred log records here. Do we need parallelization?

Reply:
Indeed, parallelization is not really needed here (and in the overwhelming majority of cases we'll have a single log anyway).